.github/workflows/dependabot-rebase.yml

```diff
@@ -13,6 +13,7 @@ on:
 jobs:
   dependabot-rebase:
```

Comment:
Move this job to `autoupdate.yml`

will need to remove `dependabot-rebase.yml` from:
- [ ] db
- [ ] build-deps

OR perhaps add a way to delete specified files from a repo automatically in `global-replicator.yml`